clash-prelude/tests/Clash/Tests/BitVector.hs

```diff
@@ -27,8 +27,8 @@ import qualified Test.Tasty.QuickCheck as Q
 import Control.Applicative (liftA2)
 #endif
 import Clash.Prelude
-  (Bit, high, low, bitPattern, type (<=), type (-), natToInteger, msb, bLit)
```

Comment:
If `bLit` is exposed from the prelude, I would also expect these new functions to be as well

Reply:
Moving the import was mostly so the line fit better. For some reason the internal module was imported even though the Prelude re-exported it.

Reply:
Exported from Prelude!